internal/seclang/parser.go

```diff
@@ -135,8 +135,7 @@ func (p *Parser) FromString(data string) error {
 
 func (p *Parser) evaluateLine(l string) error {
 	if l == "" || l[0] == '#' {
-		// at this point we should not receive an empty line or a commented line
-		return errors.New("invalid line")
+		panic("invalid line")
```

Comment:
This is a programming bug in coraza in a private function, panic is more appropriate than error

Reply:
Makes sense.